src/alvin0319/CustomItemLoader/CustomItemManager.php

```diff
@@ -117,7 +117,7 @@ public function registerItem($item) : void{
 
 			$this->itemTypeEntries[] = new ItemTypeEntry($item->getProperties()->getNamespace(), $runtimeId, true);
 
-			$this->packetEntries[] = new ItemComponentPacketEntry($item->getProperties()->getNamespace(), new CacheableNbt($item->getProperties()->getNbt()));
+			$this->packetEntries[] = new ItemComponentPacketEntry($item->getProperties()->getNamespace(), $item->getProperties()->getNbt());
```

Comment:
This is not correct. ItemComponentPacketEntry requires wrapped NBT by CachableNbt.

https://github.com/pmmp/BedrockProtocol/blob/e777b5f19946d878c6286a65877e748cf0c57079/src/types/ItemComponentPacketEntry.php#L32-L35

Reply:
excuse me i have Pocketmine 4.0.0-BETA7